libtelemetry/src/full/java/com/mapbox/android/telemetry/MapboxTelemetry.java

```diff
@@ -165,7 +165,7 @@ public boolean removeAttachmentListener(AttachmentListener listener) {
     return attachmentListeners.remove(listener);
   }
 
-  boolean isQueueEmpty() {
+  public boolean isQueueEmpty() {
```

Comment:
can we not expose public apis that are meant purely for testing?

Reply:
Yeah, will fix later.